draft-ietf-httpapi-idempotency-key-header.md

```diff
@@ -107,6 +107,8 @@ Client
 
 Clients of HTTP API requiring idempotency, SHOULD understand the idempotency related requirements as published by the server and use appropriate algorithm to generate idempotency keys.
 
+Clients MAY choose to send an Idempotency-Key field with any valid random sf-string to indicate the user's intent is to only perform this action once. Without a priori knowledge, a general client cannot assume the server will respect this request.
```

Comment:
Thanks @darrelmiller. We have not introduced these two terms : "user" and "general client".

Reply:
That's a good call out. I'll see if I can change these words to align with terms that you have already used.

Reply:
```suggestion
Clients MAY choose to send an Idempotency-Key field with any valid random sf-string to indicate the intent is to only perform this action once. Without a priori knowledge, any given client cannot assume the server will respect this request.
```
@sdatgit  How is that?